### PR TITLE
:milestone 2 components and redux store

### PR DESCRIPTION
**Pull request by firmoholanda**
in this pr, we implemented the folowing:

- prepare the directory structure using: components, containers, actions and reducers;
- implement the App component with BooksList and BooksForm;
- implement the BooksList component;
- implement the Book component;
- implement the BooksForm component;
- define the Redux actions and reducers
- initialize the Redux store.

### CODE REVIEW

File: src/components/BooksList.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```
Comment (bellom) [inline src/components/BooksList.js]: This file should also be in `src/container` dir

File: src/components/BooksList.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+import PropTypes from 'prop-types';
+
```
Comment (bellom) [inline src/components/BooksList.js]: We need to connect to the redux store and fetch the books using `mapStateToProps`
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896827/

File: src/containers/App.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```
Comment (bellom) [inline src/containers/App.js:1]: we need to re-arrange some files.
let's work on the file dir.
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896827/

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi Team :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**Please don't us move away from the instructions stated in the pathwright, know that all students are bind to make use of the project requirements.**

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Comment by Cyrus-Kiprop** [comment]
### Thanks for your wonderful review @bellom. I have implemented the requested changes in the updated pull request. They include;
- Hook BooksList component into the redux store with the help of mapStateToProps.
- Update the contents of components and container directory.

**Review by Oghenebrume50 — APPROVED** [review]
### APPROVED

Hi @firmoholanda @Cyrus-Kiprop awesome job team, this looks good to me, please move on to the next milestone

**This is approved** :+1: 

**RAPHAEL NORIODE** :mask: STAY_SAFE

**Pull request merged**